vendor/tektelic/T000489x.yaml

```diff
@@ -0,0 +1,90 @@
+name: Kona All-in-One Smart Room Sensor
+description: Multi-purpose LoRaWAN IoT sensor packed into a very small form factor
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: 'D'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: 'D'
+    numeric: 1
+    # Corresponding hardware versions (optional)
+    hardwareVersions:
+      - '1'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+    profiles:
+      EU863-870:
+        id: SENBNNEU868
```

Comment:
Profile IDs must be lowercase, max 36 chars alphanumeric and with dashes only

You can make that restriction documentation part of this pull request

Reply:
I did this already